cli/project/files.go

```diff
@@ -130,6 +139,26 @@ func GetStateboardEntrypointPath(ctx *context.Ctx) string {
 	return filepath.Join(ctx.Running.AppDir, ctx.Running.StateboardEntrypoint)
 }
 
+func getFlag(conf map[string]interface{}, opts FlagOpts) (bool, error) {
+	var flag bool
+
+	if value, found := conf[opts.ConfSectionName]; found {
+		var ok bool
+		if flag, ok = value.(bool); !ok {
+			return false, fmt.Errorf("Stateboard value should be `true` or `false`")
+		}
+
+		if opts.SpecifiedFlag != flag && flag == false {
```

Comment:
It can be done simplier, look at `getPath` function

* if user specified flag - we just use this value (opts.SpecifiedFlag != nil)
* otherwise we look at `.cartridge.yml`, and if there is no value 
* use default value

The main problem is that default boolean value is `false`, we can't check if user specified `--stateboard=false` (if he wants to override `stateboard: true` value from `.cartridge.yml`). The simpliest solution is to make `ctx.Running.WithStateboard` a pointer to `bool`.